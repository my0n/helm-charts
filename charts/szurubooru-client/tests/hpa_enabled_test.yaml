suite: test hpa
templates:
  - hpa.yaml
tests:
  - it: should match snapshot
    chart:
      version: 1.0.0
      appVersion: 1.0.0
    set:
      autoscaling.enabled: true
    asserts:
      - matchSnapshot: {}