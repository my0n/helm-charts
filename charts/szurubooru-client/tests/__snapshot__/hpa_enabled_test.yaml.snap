should match snapshot:
  1: |
    apiVersion: autoscaling/v2beta1
    kind: HorizontalPodAutoscaler
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: szurubooru-client
        app.kubernetes.io/version: 1.0.0
        helm.sh/chart: szurubooru-client-1.0.0
      name: RELEASE-NAME-szurubooru-client
    spec:
      maxReplicas: 100
      metrics:
      - resource:
          name: cpu
          targetAverageUtilization: 80
        type: Resource
      minReplicas: 1
      scaleTargetRef:
        apiVersion: apps/v1
        kind: Deployment
        name: RELEASE-NAME-szurubooru-client
